library/src/extra/rocsparse_csrgeam.cpp

```diff
@@ -22,661 +22,181 @@
-                                                                    beta,
-                                                                    csr_row_ptr_B,
-                                                                    csr_col_ind_B,
-                                                                    csr_val_B,
-                                                                    csr_row_ptr_C,
-                                                                    csr_col_ind_C,
-                                                                    csr_val_C,
-                                                                    idx_base_A,
-                                                                    idx_base_B,
-                                                                    idx_base_C);
-    }
-
-    template <typename I, typename J, typename T>
-    static rocsparse_status csrgeam_dispatch(rocsparse_handle             handle,
-                                             rocsparse_operation          trans_A,
-                                             rocsparse_operation          trans_B,
-                                             int64_t                      m,
-                                             int64_t                      n,
-                                             const T*                     alpha_device_host,
-                                             const rocsparse_mat_descr    descr_A,
-                                             int64_t                      nnz_A,
-                                             const T*                     csr_val_A,
-                                             const I*                     csr_row_ptr_A,
-                                             const J*                     csr_col_ind_A,
-                                             const T*                     beta_device_host,
-                                             const rocsparse_mat_descr    descr_B,
-                                             int64_t                      nnz_B,
-                                             const T*                     csr_val_B,
-                                             const I*                     csr_row_ptr_B,
-                                             const J*                     csr_col_ind_B,
-                                             const rocsparse_mat_descr    descr_C,
-                                             T*                           csr_val_C,
-                                             const I*                     csr_row_ptr_C,
-                                             J*                           csr_col_ind_C,
-                                             const rocsparse_spgeam_descr descr)
+    typedef rocsparse_status (*csrgeam_t)(rocsparse_handle             handle,
+                                          const rocsparse_spgeam_descr descr,
```

Comment:
Is the SpGEAM descr used?

Reply:
Yes? Are you looking at outdated files maybe?

Reply:
Actually you are correct. Fixed now.